.gitignore

```diff
@@ -8,6 +8,8 @@
 *.so
 *.dylib
 
+*.golden -text
```

Comment:
Random question as I review - is this correct with the space in here? (before `-text`)